package.json

```diff
@@ -1,17 +1,12 @@
 {
     "name": "web3.js",
     "private": true,
-    "version": "1.2.9",
```

Comment:
👍

Reply:
why are we removing the version?

Reply:
There is no need for the package version in root node_modules as lerna doesn’t use it and doesn’t update it with `lerna version`